.rubocop.yml

```diff
@@ -47,6 +47,9 @@ Layout/SpaceInsideHashLiteralBraces:
   Exclude:
     - 'views/**/*.erb'
 
+RSpec/AnyInstance:
```

Comment:
there was not really any good way around this that I know of, so no need to say: "yeah break the rules" before breaking them, I instead remove the rule.